modules/cluster-logging-deploy-subscription.adoc

```diff
@@ -59,7 +59,13 @@ metadata:
     openshift.io/cluster-logging: "true"
     openshift.io/cluster-monitoring: "true"
 ----
-<1> You must specify the `openshift-operators-redhat` namespace.
+<1> You must specify the `openshift-operators-redhat` namespace. To prevent 
+possible conflicts with metrics, you should configure the Prometheus Cluster 
+Monitoring stack to scrape metrics from the `openshift-operators-redhat` 
+namespace and not the `openshift-operators` namespace. The `openshift-operators` 
+namespace can contain community operators, which are untrusted and could publish 
+a metric with the same name as an {product-title} metric, which would cause 
+conflicts.   
```

Comment:
guessing people will ask "how do you configure cluster monitoring to scrape X and not Y" to which the answer is "via labels on the namespace" (the labels that are seen here).

maybe worth a `<2>` on the cluster-monitoring labels to explain that that is the purpose of the label.

Reply:
@bparees I added an item for the `openshift.io/cluster-monitoring: "true"` label. 
What does the `openshift.io/cluster-logging: "true"` label do?

See d7beed4a363ab9cb5054782554ca009422167ad6

Reply:
I actually do not know..  @jcantrill ?

only the cluster-monitoring label should be needed for metric scraping.

Reply:
> I actually do not know.. @jcantrill ?
> 
`git blame` shows it in a dev deployment script as part of https://github.com/openshift/elasticsearch-operator/pull/76  I know of no use for the label.  I would suggest dropping it from the docs. 

> only the cluster-monitoring label should be needed for metric scraping.